src/java.base/share/classes/javax/crypto/CryptoPolicyParser.java

```diff
@@ -42,8 +42,9 @@
  * JCE will be used.
  *
  * The jurisdiction policy file has the same syntax as JDK policy files except
- * that JCE has new permission classes called javax.crypto.CryptoPermission
- * and javax.crypto.CryptoAllPermission.
+ * that JCE has new permission classes called
+ * <code>javax.crypto.CryptoPermission</code> and
+ * <code>javax.crypto.CryptoAllPermission</code>.
```

Comment:
Use {@code } instead of <code></code> for all of the javadoc?

Reply:
At least for new changes, I prefer to use new notations.. If its easier to replace all, that's fine with me too.